apps/api/v2/src/ee/calendars/controllers/calendars.controller.ts

```diff
@@ -133,6 +146,26 @@ export class CalendarsController {
     }
   }
 
+  @UseGuards(ApiAuthGuard)
+  @Post("/:calendar/sync")
```

Comment:
It's called "/sync" endpoint and I understand that its hard to handle apple calendar in "/save" because the requests look so different. What do you think about renaming "/sync" to "/save-credentials" ? Would be more consistent with hook calling it and functions that this endpoint invokes.

Reply:
maybe we could have `@Post("/:calendar/credentials")`

Reply:
Oh yeah this sounds good, I'll rename it to `@Post("/:calendar/credentials")` and add a comment.